### PR TITLE
Remove options resolver

### PR DESCRIPTION
**Pull request by prisis**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | yes |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR |  |

No need for it, for only one setting...

### CODE REVIEW

**Comment by Nyholm** [comment]
Im 👍 , What does @aequasi think?

**Comment by prisis** [comment]
ping @aequasi

File: src/Adapter/Chain/CachePoolChain.php [hunk 1]
```diff
@@ -44,29 +44,18 @@ class CachePoolChain implements CacheItemPoolInterface, TaggablePoolInterface, L
     /**
      * @param array $pools
      * @param array $options {
-     *
-     *     @type bool $skip_on_failure If true we will remove a pool form the chain if it fails.
-     * }
+     * @type bool $skip_on_failure If true we will remove a pool form the chain if it fails.
+     *                       }
```
Comment (Nyholm) [inline src/Adapter/Chain/CachePoolChain.php]: Was it style CI that suggested this change?

Comment (prisis) [inline src/Adapter/Chain/CachePoolChain.php]: yes https://styleci.io/analyses/XaQPnm

**Comment by Nyholm** [comment]
Thank you @prisis

**Pull request merged**